src/scss/themes/satellite.scss

```diff
@@ -590,6 +590,17 @@ $break-medium: 767px;
   font-style: normal;
 }
 
+/**
+ * ReverseHighlighting and ReverseSnippeting
+ */
+
+.ais-ReverseHighlight-highlighted,
+.ais-ReverseSnippet-highlighted {
+  background-color: none;
```

Comment:
```suggestion
```

I don't think is line is needed

Reply:
We can add `background: none` because `mark` has a default background in most browsers.